locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocationResolver.java

```diff
@@ -50,10 +50,10 @@
 
     public static final Logger log = LoggerFactory.getLogger(JcloudsLocationResolver.class);
     
-    public static final String JCLOUDS = "jclouds";
+    private static final String JCLOUDS = "jclouds";
     
-    public static final Map<String,ProviderMetadata> PROVIDERS = getProvidersMap();
-    public static final Map<String,ApiMetadata> APIS = getApisMap();
+    private static final Map<String,ProviderMetadata> PROVIDERS = getProvidersMap();
+    private static final Map<String,ApiMetadata> APIS = getApisMap();
```

Comment:
extensions might be accessing these.  if they need to be private then let's go through a deprecation cycle.